### ISSUE TITLE
Example Scene does not work

### ISSUE THREAD

**Issue by Nanorock** [Unity-Technologies/Animation-Instancing#29]
Launching the example scene just throw errors straight from the start.
In the hierarchy "Dude_low_instancing", AnimationInstancing component is missing the Prototype. 
The problem does not end here as filling in the prefab still does not work throwing further errors.
I'm currently running through the code and see if I can help

**Comment by Velo222** [comment]
I had the same thing.  I got the scene to stop throwing errors by doing the following:  Click on the "Dude_low_instancing" gameobject in the scene hierarchy (the TestScene.unity).  Under the script component "Animation Instancing (Script)" assign the "Dude_low_instancing" prefab to the Prototype slot on the script component.   The Dude prefab is located in AniInstancing>>Example>>Prefab.

If you start the game at this point, there will still be errors (something about end of stream or something).  So, you also have to click on the "AnimationInstancing" menu tab in the Unity editor and select "Animation Generator".  Generate the new animation (I forget exactly what it's called).  

Now play the game and it should run without errors.  I had to click in the scene window to get the guy to appear.  I have no idea what this test scene does, other than show one guy running.... that's as far as I've gotten.  Hope it might help.

**Comment by Velo222** [comment]
I just noticed there are two test scenes.  I got the "TestScene.unity" scene to work by doing what I posted above.  However, there is another scene called "Crowd Simulation.unity" which still does not work for me.

I am getting the error "EndOfStreamException: Failed to read past end of stream.
System.IO.BinaryReader.FillBuffer (Int32 numBytes) (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.IO/BinaryReader.cs:119)
System.IO.BinaryReader.ReadString () (at /Users/builduser/buildslave/mono/build/mcs/class/corlib/System.IO/BinaryReader.cs:486)
AnimationInstancing.AnimationManager.ReadAnimationInfo (System.IO.BinaryReader reader) (at Assets/AniInstancing/Scripts/AnimationManager.cs:235)
AnimationInstancing.AnimationManager+<LoadAnimationInfoFromAssetBundle>c__Iterator1.MoveNext () (at Assets/AniInstancing/Scripts/AnimationManager.cs:133)
UnityEngine.SetupCoroutine.InvokeMoveNext (IEnumerator enumerator, IntPtr returnValueAddress) (at C:/buildslave/unity/build/Runtime/Export/Coroutines.cs:17)"
